src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -133,13 +133,22 @@ void enumerateSnapshotRequestsByDatasetId() {
   public void getConceptChildren(CloudPlatform cloudPlatform) {
```

Comment:
It doesn't hurt to make tests `public`, but it's not necessary and sonar will flag it as an issue.
```suggestion
 void getConceptChildren(CloudPlatform cloudPlatform) {
```